server/blockchain/calypso/calypso_test.go

```diff
@@ -260,19 +318,22 @@ func TestCommand_ListSecrets(t *testing.T) {
 
 	snap := fake.NewSnapshot()
 
-	err := cmd.advertiseSmc(snap, makeStep(t, KeyArg, "dummy", RosterArg, "node:12345"))
+	err := cmd.advertiseSmc(snap, makeStep(t, SmcPublicKeyArg, "dummy", RosterArg, "node:12345"))
```

Comment:
same, unless I'm missing something, I'd rather have a `fake.NewSnapshot()` for each call.

Reply:
it doesn't mean my original code was good code 😅